Backpack/src/main/java/net/skyscanner/backpack/calendar2/list/CalendarCellHeaderHolder.kt

```diff
@@ -48,8 +49,16 @@ internal class CalendarCellHeaderHolder(
   override fun bind(model: CalendarCell.Header) {
     month.text = model.title
     btnSelectWholeMonth.apply {
-      model.selectWholeMonthLabel?.let { text = it }
-      isVisible = !model.selectWholeMonthLabel.isNullOrEmpty()
+      when {
```

Comment:
isVisible = monthSelectionMode is CalendarParams.MonthSelectionMode.SelectWholeMonth
isEnabled = selectionMode !is CalendarParams.SelectionMode.Disabled

Reply:
Applied 👍